ghost/core/core/server/data/schema/schema.js

```diff
@@ -404,7 +404,8 @@ module.exports = {
         created_at_ts: {type: 'bigInteger', nullable: false},
         created_at: {type: 'dateTime', nullable: false},
         author_id: {type: 'string', maxlength: 24, nullable: true, references: 'users.id', cascadeDelete: false, constraintName: 'post_revs_author_id_foreign'},
-        title: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 255}}}
+        title: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 255}}},
+        post_status: {type: 'string', maxlength: 50, nullable: true, validations: {isIn: [['draft', 'published', 'scheduled']]}}
```

Comment:
should we add 'sent' as an available status? I think if they send as an email but don't publish, the status will be 'sent', which we prob want to indicate on the revisions as well.

Reply:
oh yea looks like you've got it in the migration, just not in the schema